.github/workflows/conformance-suites.yml

```diff
@@ -1,5 +1,8 @@
 name: Run Xbrl Conformance Suites
-on: pull_request_target
+on:
+  pull_request: pull_request_target
```

Comment:
I don't think this means "pull_request_target".  Maybe
```
on:
  pull_request_target:
  release:
    types: [created]
```

Reply:
It would be best to test this against a fork before merging